### PR TITLE
weather: Shorter names, repo selection and fixes

### PR DESCRIPTION
**Pull request by marusak**
As always, preview: https://mmarusak.fedorapeople.org/tests.html

I have fixed mine DB in ^. DB we have in production is a bit broken. I can provide fixed DB and we can put that into images store.

### CODE REVIEW

File: tests.html [hunk 1]
```diff
@@ -245,7 +250,7 @@ <h2 id="g2-title"></h2>
                         ORDER BY run DESC
                         LIMIT 2`);
                     const links = urls[0].values.map(u => u[0]);
-                    top10.push([{name: row[0], key: "test"}, row[1].toFixed(2), links]);
+                    top10.push([{name: normalize(row[0]), key_name: row[0], key: "test"}, row[1].toFixed(2), links]);
```
Comment (martinpitt) [inline tests.html]: Any chance we can add the project here? check-application is rather vague, our current top failures are from cockpit-certificates. The new selector surely helps with that, though!

**Review by martinpitt — APPROVED** [review]
Whee, every round this becomes so much better, thank you! Let's get this in ASAP to not further collect the wrongly unified contexts.

File: tests.html (hunk 1 above)
Comment (marusak) [reply tests.html]: Good idea! Pushed additional commit. (And deployed it on my page as well for reviewing).

**Review by martinpitt — APPROVED** [review]
Cool, thank you!

**Pull request merged**